src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image-imac"
+        src="./images/imac.jpeg"
+        alt="imac 27"
+      >
+
+      <h2 class="card__name-of-product">
```

Comment:
It's okay, but class name is so long. You can make it shorter. But maybe, it's not critical.

Reply:
I'm not sure how is better:( Maybe short name will be less understanding

Reply:
It's okay, you can leave it, mentor will help if this critical

Reply:
short name is better
BEM has a problem with a long name when we combine block + element + modifier. `card__name-of-product--active` is a long name so it's better to keep names short and understandable